From b3d6dfe44a76c4f3bf00eb901c1844b12300c66e Mon Sep 17 00:00:00 2001
Message-Id: <b3d6dfe44a76c4f3bf00eb901c1844b12300c66e.1570736137.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Mon, 29 Apr 2019 04:28:23 -0400
Subject: [PATCH] pthread.h: manually align parameter lists
To: Git Mailing List <git@vger.kernel.org>

In previous patches, extern was mechanically removed from function
declarations without care to formatting, causing parameter lists to be
misaligned. Manually format changed sections such that the parameter
lists are realigned.

Viewing this patch with 'git diff -w' should produce no output.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
I missed a step in 'dl/compat-cleanup'. This patch can be applied on the
tip of that branch.

 compat/win32/pthread.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/compat/win32/pthread.h b/compat/win32/pthread.h
index f1cfe73de9..737983d00b 100644
--- a/compat/win32/pthread.h
+++ b/compat/win32/pthread.h
@@ -51,7 +51,7 @@ typedef struct {
 } pthread_t;
 
 int pthread_create(pthread_t *thread, const void *unused,
-			  void *(*start_routine)(void*), void *arg);
+		   void *(*start_routine)(void*), void *arg);
 
 /*
  * To avoid the need of copying a struct, we use small macro wrapper to pass
-- 
2.23.0.746.g72fc0fc0b9

